src/NLog/Targets/NetworkTarget.cs

```diff
@@ -509,7 +513,7 @@ private LinkedListNode<NetworkSender> GetCachedNetworkSender(string address)
         private NetworkSender CreateNetworkSender(string address)
         {
 #if !SILVERLIGHT
-            var sender = SenderFactory.Create(address, MaxQueueSize, SslProtocols);
+            var sender = SenderFactory.Create(address, MaxQueueSize, SslProtocols, TimeSpan.FromSeconds(KeepAliveTimeSeconds));
```

Comment:
Maybe push it down as far as possible ;)

(timespan instead of secs)

Reply:
Prefer to keep it as seconds in the XML

Reply:
The timespan is for times, not ints ;)

isn't 1 minute a valid value? Then `0:1:00` is better then 60 IMO

Reply:
AFAIK all timings in C# are timespans?

Reply:
You enter 30, and you get 30 days.

Reply:
Didn't say I didn't like TimeSpan. I just think the TimeSpan-String-Format is horrible when interval is seconds.

Reply:
OK fair enough :) sec in XML then